src/components/ScenarioComponents/CardsComponents/MainCard/CardTooltip.tsx

```diff
@@ -17,39 +18,34 @@ interface CardTooltipProps {
   color: string;
 
   /** The title of the card. */
-  index: number;
-
-  /** A boolean indicating whether the scenario is active. */
-  activeScenario: boolean;
-
-  /** An array of active scenarios. */
-  activeScenarios: number[] | null;
+  id: string;
 
   /** The number of the selected scenario. */
-  numberSelectedScenario: number | null;
+  setSelected: Dispatch<{id: string; state: boolean}>;
```

Comment:
This comment doesn't match the member below 🤔
```suggestion
  /** A function to set the selected scenario. */
  setSelected: Dispatch<{id: string; state: boolean}>;
```